lib/capybara/node/element.rb

```diff
@@ -429,17 +430,17 @@ def drop(*args)
 
       ##
       #
-      # Scroll the page or element
+      # Scroll the page or element.
       #
-      # Scroll the page or element to its top, bottom or middle
       # @overload scroll_to(position, offset: [0,0])
+      #   Scroll the page or element to its top, bottom or middle.
```

Comment:
[question] Is `middle` a mistake of `center`? The parameter description is below: ⬇️ 

```ruby
# @param [:top, :bottom, :center, :current] position
```

Reply:
In the text description center and middle are generally equivalent. If you want to change it to center to match the parameter value I don't have an issue with that.

Reply:
OK, I keep as it is. 👍